ui/src/app/components/runtime-configuration-panel/start-stop-runtime-button.tsx

```diff
@@ -1,31 +1,56 @@
 import * as React from 'react';
 
-import { RuntimeStatus } from 'generated/fetch';
+import { AppStatus, RuntimeStatus } from 'generated/fetch';
 
+import {
+  fromRuntimeStatus,
+  fromUserAppStatus,
+  UIAppType,
+  UserEnvironmentStatus,
+} from 'app/components/apps-panel/utils';
 import { Clickable } from 'app/components/buttons';
 import { FlexRow } from 'app/components/flex';
 import { TooltipTrigger } from 'app/components/popups';
 import colors, { addOpacity } from 'app/styles/colors';
-import { DEFAULT, switchCase } from 'app/utils';
-import {
-  RuntimeStatusRequest,
-  useRuntimeStatus,
-} from 'app/utils/runtime-utils';
+import { cond, DEFAULT, switchCase } from 'app/utils';
 import computeError from 'assets/icons/compute-error.svg';
 import computeNone from 'assets/icons/compute-none.svg';
 import computeRunning from 'assets/icons/compute-running.svg';
 import computeStarting from 'assets/icons/compute-starting.svg';
 import computeStopped from 'assets/icons/compute-stopped.svg';
 import computeStopping from 'assets/icons/compute-stopping.svg';
 
+interface StatusInfo {
+  status: AppStatus | RuntimeStatus;
+  onPause: () => Promise<any>;
+  onResume: () => Promise<any>;
+  appType: UIAppType;
+}
+
 export const StartStopRuntimeButton = ({
```

Comment:
Yes, we should not refer to runtimes specifically, if this is now generic.  We should also update other variables, text and ID, like:
```
altText: 'Runtime running, click to pause',
        iconSrc: computeRunning,
        dataTestId: 'runtime-status-icon-running',
        onClick: onPause,
```
```
altText: 'Environment running, click to pause',
        iconSrc: computeRunning,
        dataTestId: 'environment-status-icon-running',
        onClick: onPause,
```
etc.

But also: I wonder if all this work is needed at this time.  Because this button won't be operational for Cromwell MVP, right?  Maybe we should revert these changes and save the work for the real Cromwell Config Panel.

Reply:
Was not a hard refactor. Was able to do it decently quickly.